Backpack/src/androidTest/java/net/skyscanner/backpack/skeleton/BpkBodyTextSkeletonTest.kt

```diff
@@ -0,0 +1,44 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.skeleton
+
+import android.graphics.drawable.ColorDrawable
+import android.widget.RelativeLayout
+import androidx.test.annotation.UiThreadTest
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import net.skyscanner.backpack.R
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkBodyTextSkeletonTest {
+
+  @Test
+  @UiThreadTest
+  fun test_bodyTextSkeleton() {
```

Comment:
we generally have snapshot tests for any UI checks (see https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md#snapshot-testing)

Reply:
Thanks. Actually I have added UI tests, perhaps we'd better to delete the unit tests for those UI components without logic.